CONTRIBUTING.md

```diff
@@ -82,64 +82,100 @@ $ make build-dist
 
 ### Running linting/tests
 
-You can run lint via:
+#### You can run lint via:
```

Comment:
Don't know if we want to align the language in h4 subtitles. e.g. We can simplify put "Lint" here?

Reply:
My idea was to add a bit of structure to all these lines. But we can rethink it. :)